lib/chai/core/assertions.js

```diff
@@ -3134,14 +3142,29 @@ function isSubsetOf(subset, superset, cmp, contains, ordered) {
  * @api public
  */
 
+Assertion.addMethod('iterable', function(msg) {
+  if (msg) flag(this, 'message', msg);
+  var obj = flag(this, 'object')
+    , flagMsg = flag(this, 'message')
+    , ssfi = flag(this, 'ssfi');
+
+  if (typeof obj === 'string' || !obj[Symbol.iterator]) {
+    throw new AssertionError(
+      `${flagMsg}: expected ${_.inspect(obj)} to be an iterable`,
+      undefined,
+      ssfi
+    );
+  }
+});
```

Comment:
This needs to have a JSDoc comment and needs to move so `members` has the correct JSDoc comment associated with it.